contracts/LiquidPledgingBase.sol

```diff
@@ -38,10 +40,10 @@ contract LiquidPledgingBase is Owned {
     uint constant MAX_INTERPROJECT_LEVEL = 20;
 
     enum PledgeAdminType { Giver, Delegate, Project }
-    enum PaymentState { Pledged, Paying, Paid }
+    enum PaymentState { Pledged, Paying, Paid } //TODO Very confusing with PaymentStatus rename to PledgeState??????????????
```

Comment:
I agree

Reply:
Sweet, I don't know if it needs to delay the launch but i would like to leave the comments in and at least make it part of the next develop branch